superset/assets/src/visualizations/MapBox/MapBox.jsx

```diff
@@ -98,14 +103,14 @@ class MapBox extends React.Component {
           isDragging={isDragging}
           width={width}
           height={height}
-          locations={Immutable.fromJS(this.clusters)}
+          locations={Immutable.fromJS(clusters)}
           dotRadius={pointRadius}
           pointRadiusUnit={pointRadiusUnit}
           rgb={rgb}
           globalOpacity={globalOpacity}
           compositeOperation={'screen'}
           renderWhileDragging={renderWhileDragging}
-          aggregatorName={aggregatorName}
+          aggregation={customMetric ? aggregatorName : undefined}
```

Comment:
NIT: I think we generally prefer `null` over `undefined` though effectively it's the same

Reply:
Yep, can change this to `null`.